region/crateria/central/Climb.json

```diff
@@ -502,6 +503,25 @@
         "Enemies will not hurt Samus since they are non-global and also off-camera."
       ]
     },
+    {
+      "link": [2, 5],
+      "name": "Transition with Stored Fall Speed",
+      "entranceCondition": {
+        "comeInWithStoredFallSpeed": {
+          "fallSpeedInTiles": 2
+        }
+      },
+      "requires": [
+        "ScrewAttack",
+        "Grapple"
+      ],
```

Comment:
Short-charging (in-room) would also be an option for getting across, with either low or high fall speed, in either direction.